claat/render/md.go

```diff
@@ -232,3 +238,33 @@ func (mw *mdWriter) header(n *types.HeaderNode) {
 		mw.writeBytes(newLine)
 	}
 }
+
+func (mw *mdWriter) grid(n *types.GridNode) {
+  mw.newBlock()
+  mw.writeString(`<table markdown="1">`)
+  for _, r := range n.Rows {
+    mw.writeString("\n<tr>")
+    for _, c := range r {
+      mw.writeFmt(`<td colspan="%d" rowspan="%d">`, c.Colspan, c.Rowspan)
+      mw.write(c.Content.Nodes...)
+      mw.writeString("</td>")
+    }
+    mw.writeString("\n</tr>")
+  }
+  mw.writeString("</table>")
+}
+
+func (mw *mdWriter) infobox(n *types.InfoboxNode) {
+  mw.newBlock()
+  mw.writeString(`<aside markdown="1" class="`)
+  mw.writeEscape(string(n.Kind))
+  mw.writeString(`">`)
+  mw.write(n.Content.Nodes...)
+  mw.writeString("</aside>")
+}
+
+func (mw *mdWriter) youtube(n *types.YouTubeNode) {
+  mw.newBlock()
+  mw.writeFmt("<iframe src=\"https://www.youtube.com/embed/%q\" ", n.VideoID)
```

Comment:
1. you can use backquotes. it reads better:

    mw.writeFmt(`<iframe src="..." `, n.VideoID)

2. the "%q" wraps string in quotes. I don't think it's what you want. you probably meant "%s"
3. are we sure n.VideoID is safe? if in doubt, I'd use [url.PathEscape](https://golang.org/pkg/net/url/#PathEscape)